src/app/onion/shared/submit/submit.component.scss

```diff
@@ -2,9 +2,10 @@
 
 .submit {
   max-width: 800px;
-  min-height: 550px;
+  height: 85vh;
```

Comment:
I think we should use a mix of ```max-height``` and ```height``` here instead.  That way the popup isn't too big on larger screens.

Reply:
I can give it a shot but the popup doesn't really do what you would expect it to do, even set to 85vh it is still kinda cut off on the top and bottom when zoomed in far enough to experience the bug.

Reply:
ok I added a max-height of 550px